CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add default assigners permission [#477](https://github.com/rokwire/core-building-block/issues/477)
 
 ### Fixed
+- Fix system account initialization [#594](https://github.com/rokwire/core-building-block/issues/594)
```

Comment:
It should be in the `Fixed` section but under `[Unreleased]`

Reply:
Thanks for the catch @petyos!